src/openapi.yaml

```
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   title: Open Forms API
-  version: 2.0.0
+  version: 2.1.0-beta.0
```

```suggestion
  version: 2.1.0-alpha.2
```

I think the API version should always be the same as the software, even if we think the API will not get any more features and the software will.

---

missed this one, it should be alpha in this case. However, the backend version itself can be newer than the API version if the changes do not affect the API spec at all - e.g. API version 2.1.0 and backend version 2.1.5.

We are documenting the version of the API spec, not the version of the API per the OAS specification:

> REQUIRED. The version of the OpenAPI **document** (which is distinct from the [OpenAPI Specification version](https://swagger.io/specification/#oas-version) or the API implementation version).

(emphasis mine, see https://swagger.io/specification/#info-object)